demos/ns-inputter--radio/index.html

```diff
@@ -0,0 +1,16 @@
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+</head>
+
+<body>
+  <main class="ndsn">
+    <ns-inputter mask="" separator="" validation="[&quot;isRequired&quot;]" heading="Are you a British Gas customer?" helper="" name="yes" value="">
```

Comment:
Attributes `mask` and `separator` are not available for radio buttons.
```suggestion
    <ns-inputter validation="[&quot;isRequired&quot;]" heading="Are you a British Gas customer?" helper="" name="yes" value="">
```